core/templates/dev/head/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -120,6 +120,19 @@ angular.module('oppia').factory('ResponsesService', [
       }
     };
 
+    var _reduceRuleIndexByOne = function(answerGroupIndex, ruleIndex) {
+      if (_answerGroups[answerGroupIndex].rules[ruleIndex].inputs.x > 0) {
```

Comment:
Just to check, in practice this condition is never false, right? (because of earlier condition on index which would need to be negative if this is called).